pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -450,7 +452,7 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				// was allocated.
 				_, _, err := m.containerMap.GetContainerRef(containerID)
 				if err == nil {
-					klog.V(4).InfoS("ReconcileState: ignoring terminated container", "pod", klog.KObj(pod), "containerID", containerID)
+					klog.V(4).InfoS("ReconcileState: terminated container ignored", "pod", klog.KObj(pod), "containerID", containerID)
```

Comment:
nit: `ignoring terminated container` ?
your call :)

Reply:
Was attempting to follow the recommendations in the log message styling guidelines [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#message-style-guidelines) and change to past tense :) If you prefer to leave it the way it is, I don't mind :)

Reply:
If you didn't change already, leave as it is. It's really a nit